tsdb/track.json

```diff
@@ -3,14 +3,42 @@
 {
   "version": 2,
   "description": "metricbeat information for elastic-app k8s cluster",
+  {%- if ingest_mode is defined and ingest_mode == "data_stream" %}
+  "composable-templates": [
+    {
+      "name": "tsdb-template",
+      "index-pattern": "k8s*",
+      "delete-matching-indices": true,
+      "template": "index-template.json"
+    }
+  ],
+  "data-streams": [
+    {
+      "name": "k8s"
+    }
+  ],
+  {%- else %}
   "indices": [
     {
       "name": "tsdb",
       "body": "index.json"
     }
   ],
+  {%- endif %}
   "corpora": [
-    {%- if ingest_order is defined and ingest_order == "sorted" %}
+    {%- if ingest_mode is defined and ingest_mode == "data_stream" %}
```

Comment:
For those following along, we'll likely switch everything to the deduped version in a follow up.